staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -1360,6 +1360,44 @@ const (
 	// DisablePCIHole64 indicates that the 64-Bit PCI hole should be disabled on a VirtualMachineInstance.
 	// This annotation might be deprecated in the future if we decided to add a struct for it.
 	DisablePCIHole64 string = "kubevirt.io/disablePCIHole64"
+
+	// AllowAccessClusterServicesNPLabel indicates if the virt components requires access to the cluster services in
+	// case of usage of strict network policy.
+	// This label can be used as pod selector to create a NP, allowing access to the cluster services (apiserver/dns).
+	// An example of a NP might be:
```

Comment:
This is a bit unclear, here's a clarification attempt (without fully understanding the label, might be wrong)
```suggestion
	// AllowAccessClusterServicesNPLabel is a pod label to be set by virt- components to indicate that they require access to cluster services otherwise blocked by the strict network policy (NP).
	// This label is then used as pod selector to create a NP to give the pod access to cluster services (apiserver/dns).
	// An example of a NP might be:
```